app/controllers/users/invitations_controller.rb

```diff
@@ -63,7 +63,7 @@ def authenticate_inviter!
   end
 
   def delete_user_record
-    invited_user.destroy!
+    invited_user.destroy! unless invited_user.nil?
```

Comment:
could shorten to '''invited_user&.destroy!```

Reply:
I love a `&.` but feel the existing is more readable so will leave as is if you're ok with that... not merging yet until the deploy issue is resolved.